app/src/androidTest/java/com/github/h3lp3rs/h3lp/AwaitHelpActivityTest.kt

```diff
@@ -168,4 +178,46 @@ class AwaitHelpActivityTest {
 
         return intent
     }
-}*/
\ No newline at end of file
+
+    @Test
+    fun getsNotifiedWhenHelpIsComing() {
+        // Forge the right intent
+        val helpId = 1
+        val bundle = Bundle()
+        bundle.putInt(EXTRA_EMERGENCY_KEY, helpId)
+        bundle.putBoolean(EXTRA_CALLED_EMERGENCIES, true)
+        bundle.putStringArrayList(EXTRA_NEEDED_MEDICATION, arrayListOf(EPIPEN))
+        val intent = Intent(
+            getApplicationContext(),
+            AwaitHelpActivity::class.java
+        ).apply {
+            putExtras(bundle)
+        }
+        // Setup the database accordingly
+        val emergencyDb = MockDatabase()
+        val skills = HelperSkills(true, true, true, true,
+            true, true)
+        val emergency = EmergencyInformation(helpId.toString(), 2.0, 2.0, skills,
+            ArrayList(listOf("Epipen")), Date(), null, ArrayList())
+        emergencyDb.setObject(helpId.toString(), EmergencyInformation::class.java, emergency)
+        EMERGENCIES.db = emergencyDb
+        // Simulate arrival on await page after calling for help
+        launch<AwaitHelpActivity>(intent).use {
+            // Nobody coming
+            onView(withId(R.id.incomingHelpersNumber)).check(matches(withText("")))
+            // One person is coming
+            val helper1 = Helper(USER_TEST_ID + 1, 2.0, 2.0)
+            val withHelpers = emergency.copy(helpers = ArrayList(listOf(helper1)))
+            emergencyDb.setObject(helpId.toString(), EmergencyInformation::class.java, withHelpers)
+            onView(withId(R.id.incomingHelpersNumber)).check(matches(withText("1 person is coming to help you")))
```

Comment:
maybe use string ressource for this then when we changed it the test won't break

Reply:
Will do, it has not been done in the code but I should have changed it.